app/src/main/res/values-nl/strings.xml

```diff
@@ -143,9 +143,9 @@
 
         De galerij kan ook worden gebruikt voor het bekijken van afbeeldingen of video\'s vanuit andere apps, om bijlagen toe te voegen in e-mail, etc. Perfect voor dagelijks gebruik.
 
-        De permissie voor het uitlezen van vingerafdrukken is benodigd voor het vergendelen van verborgen items of de gehele app.
+        Het toegangsrecht voor het uitlezen van vingerafdrukken is benodigd voor het vergendelen van verborgen items of de gehele app.
```

Comment:
Either "permissie" or "machtiging" (like in Play Store)

Reply:
It says "toegangsrecht" here, which is also how it's translated in most other software using Dutch (Netherlands) translation. "permissie" is a Flemish word.

Reply:
It's 100% Dutch, but it's fine if there's a more official translation in the Android context. Just curious where "here" is. Never seen "toegangsrecht" in Android before.

Reply:
I've asked a few family members of mine, who studied Dutch at the university, and they say that "permissie" is not official Dutch but rather an Anglicism.